lib/preload.js

```diff
@@ -1,6 +1,3 @@
-// Get some logs when things go horribly wrong
-require('./sentry')
```

Comment:
I dropped this because Electron 20 doesn't allow preload scripts to access the node environment without the `nodeIntegration` flag, which is strongly discouraged:  https://www.electronjs.org/docs/latest/tutorial/security 

There's probably a way to get Sentry integration working in a safe way. However, we added Sentry support for our own uses, and we aren't using it anymore. So unless and until someone else has a need for it, I figured it was easiest to just drop it for now.

Reply:
There is a section in the README about Sentry, you should remove that too.